scala/scala.bzl

```diff
@@ -233,25 +233,15 @@ def _compile_or_empty(ctx, jars, srcjars, buildijar):
     return struct(ijar=ijar, class_jar=ctx.outputs.jar)
 
 def _build_deployable(ctx, jars):
-  cmd = "rm -rf {out}_tmp\n"
-  cmd += "mkdir -p {out}_tmp\n"
-  for jar in jars:
-    cmd += "unzip -o {jar} -d {{out}}_tmp >/dev/null\n".format(jar=jar.path)
-  cmd += "{java} -jar {jar} -m {manifest} {out} {out}_tmp\n"
-  cmd += "rm -rf {out}_tmp\n"
-
-  cmd = cmd.format(
-      out=ctx.outputs.deploy_jar.path,
-      jar=_get_jar_path(ctx.files._jar),
-      java=ctx.file._java.path,
-      manifest=ctx.outputs.manifest.path)
+  args = ["-m", ctx.outputs.manifest.path, ctx.outputs.deploy_jar.path]
```

Comment:
what does -m stand for? Obviously a noob question but maybe should be a constant? a bit of a magic symbol?